### ISSUE TITLE
mirage-xen-ocaml uninstall is broken

### ISSUE THREAD

**Issue by avsm** [mirage/mirage-platform#129]
```
[WARNING] failure in package uninstall script, some files may remain:
          # opam-version 1.2.1
          # os           linux
          # command      make xen-ocaml-uninstall
          PREFIX=/home/travis/.opam/system
          # path         /home/travis/.opam/system/build/mirage-xen-ocaml.2.3.0
          # compiler     system (4.02.1)
          # exit-code    2
          # env-file
          /home/travis/.opam/system/build/mirage-xen-ocaml.2.3.0/mirage-xen-ocaml-18906-548b09.env
          # stdout-file
          /home/travis/.opam/system/build/mirage-xen-ocaml.2.3.0/mirage-xen-ocaml-18906-548b09.out
          # stderr-file
          /home/travis/.opam/system/build/mirage-xen-ocaml.2.3.0/mirage-xen-ocaml-18906-548b09.err
          ### stdout ###
          # cd xen-ocaml && make uninstall
          # make[1]: Entering directory
          `/home/travis/.opam/system/build/mirage-xen-ocaml.2.3.0/xen-ocaml'
          # ./uninstall.sh
          # make[1]: Leaving directory
          `/home/travis/.opam/system/build/mirage-xen-ocaml.2.3.0/xen-ocaml'
          ### stderr ###
          # [...]
          # +prefix=
          # +[  =  ]
          # +opam config var prefix
          # +prefix=/home/travis/.opam/system
          # +odir=/home/travis/.opam/system/lib
          # +rm -f /home/travis/.opam/system/lib/pkgconfig/mirage-xen-ocaml.pc
          # +rm -f /home/travis/.opam/system/lib/mirage-xen-ocaml
          # rm: cannot remove `/home/travis/.opam/system/lib/mirage-xen-ocaml':
          Is a directory
          # make[1]: *** [uninstall] Error 1
          # make: *** [xen-ocaml-uninstall] Error 2
[WARNING] Directory /home/travis/.opam/system/lib/mirage-xen-ocaml is not
          empty, not removing
```

**Comment by hannesm** [comment]
I fixed it there https://github.com/mirage/mirage-platform/commit/f945e7deafd5f420715b4f888f708f1e23212a1a -- merged, but not released...

**Comment by avsm** [comment]
released in 2.3.1
